app/api/routes.py

```diff
@@ -8,17 +8,26 @@
 from app.utils import Paginator, standardize_response, setup_logger
 from dateutil import parser
 from datetime import datetime
+from prometheus_client import Counter, Summary
 import uuid
 
+# Metrics
+c = Counter('my_failures', 'Number of exceptions raised')
+s = Summary('request_latency_seconds', 'Length of request')
```

Comment:
Same here with regards to naming. You may have many summaries for different things, including specific endpoints or different request types.